channel.go

```diff
@@ -263,9 +267,16 @@ func TruncateWithSkipPush() func(*truncateOptions) {
 	}
 }
 
-func TruncateWithMessage(message *Message) func(*truncateOptions) {
+func TruncateWithMessage(message *Message, userID string) func(*truncateOptions) {
```

Comment:
this is a breaking change but I assume this is a breaking change in the backend side as well: https://github.com/GetStream/chat/blob/master/controllers/truncate_channel.go#L77=

Reply:
this isn't needed since `user_id` is passed inside `message` while the other is needed since we can truncate without message

Reply:
okay, I'll remove this